scripts/scil_tractogram_alter.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+
+"""
+This script is used to alter a bundle to reach specific minimum dice
+coefficient from the original bundle. The script will subsample, trim, cut,
```

Comment:
It took me a little while to understand the goal of the script. Maybe something like this for the first sentence?

`This script is used to alter a bundle while keeping it similar enough to the original version (keeping the dice
coefficient above a given threshold).`